lib/web/ui/perf_test.go

```diff
@@ -148,7 +148,7 @@ func insertServers(ctx context.Context, b *testing.B, svc services.Presence, kin
 func benchmarkGetClusterDetails(ctx context.Context, b *testing.B, site reversetunnelclient.RemoteSite, nodes int, opts ...services.MarshalOption) {
 	var cluster *Cluster
 	var err error
-	for i := 0; i < b.N; i++ {
+	for b.Loop() {
```

Comment:
The call to benchmarkGetClusterDetails is sandwiched between StartTimer/StopTimer, I think we don't need that anymore either, right?